appcelerator/titanium_mobile
# [TIMOB-15576]iOS: adding support for iOS 7 64bit iPad sim

**TESTING INSTRUCTIONS**
Code Review of the ios-sim project to be done on https://github.com/appcelerator/ios-sim/pull/6

the testing can be done only through CLI. 
- Go to any Titanium app folder in terminal or create a new app.
- "ti build -p ios -Y ipad -S 7.0 --retina --sim-64bit"
- Kill the ios simulator app. 
- "ti build -p ios -Y iphone -S 7.0 --retina --tall --sim-64bit"
- Kill the ios simulator app. 

**Make sure to kill the simulator before running the next command. Otherwise the right simulator will not be displayed**
